eng/build-job.yml

```diff
@@ -54,9 +65,8 @@ jobs:
     steps:
 
     # Install native dependencies
-    #
-    # This is only required for non-docker builds.
     - ${{ if eq(parameters.osGroup, 'OSX') }}:
+      # This is only required for non-docker builds.
```

Comment:
This comment is no longer correct as freeBSD is a non-docker build. Can you add a comment explaining why freeBSD does not need native dependencies setup atm.

Reply:
Fixed, thank you!